blueprints/sparkles-component/files/__root__/__path__/__name__.__ext__

```diff
@@ -0,0 +1,5 @@
+import Component, { tracked } from 'sparkles-component';
```

Comment:
Hmm, do you think that `tracked` should be imported by default? Will > 80% of sparkles-components need / want that?

Reply:
My thinking was yes, but it's anyone's guess. We can always revise this in future blueprints. What are your thoughts?

Reply:
TBH, I haven’t really thought too much about it. I think assuming that it will be used is fine for now, its just _slightly_ annoying that it makes a `yarn lint:js` fail immediately after generating but that’s not a big deal (I think the tests would fail also so meh)

Reply:
I played with this and changed my mind. Let's add `tracked` as a default import if the need arises